src/main/java/org/grouplens/grapht/Instantiators.java

```diff
@@ -39,6 +42,12 @@ private Instantiators() {}
      * @param inst The instance to return (must be non-null).
      * @return An instantiator that returns {@code inst}.
      */
+
+
+    public static final Logger     Logger = LoggerFactory.getLogger(Instantiator.class);
```

Comment:
We usually name our loggers `logger`, and make them private. Also, we don't need the extra space in this line, or above it.

And the JavaDoc comment above needs to be moved back down so it is right above the `ofInstance` method.